src/main/java/com/EighthLight/app/Board.java

```diff
@@ -51,6 +51,16 @@ public ArrayList getSpaces() {
         return spaces;
     }
 
+    public ArrayList<Integer> findEmptySpaces(ArrayList spaces) {
```

Comment:
move findEmptySpaces to Board class, as it has to do with the the current state of the board.